aries-site/src/examples/components/button/DosDonts/ButtonBadCTAPreview.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import { Box, Button, Header, Heading, Text } from 'grommet';
+
+export const ButtonBadCTAPreview = () => (
+  <Box gap="small" pad="small">
```

Comment:
should we set a slightly wider with on this so that it's a bit more obvious the difference between the "do" and "Dont"?

Reply:
yes!